<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ubah Produk</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_edit_row">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="inputMask">
    <string comment="&quot;00&quot;" extracomment="&quot;00&quot;"/>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_edit_name">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="inputMask">
    <string comment="&quot;00&quot;" extracomment="&quot;00&quot;"/>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_edit_merk">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="inputMask">
    <string comment="&quot;00&quot;" extracomment="&quot;00&quot;"/>
   </property>
  </widget>
  <widget class="QPushButton" name="button_edit_produk">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>310</y>
     <width>161</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Ubah Produk</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nama:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Merk:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_edit_sell_price">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="inputMask">
    <string comment="&quot;00&quot;" extracomment="&quot;00&quot;"/>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_edit_discount">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>220</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="inputMask">
    <string comment="&quot;00&quot;" extracomment="&quot;00&quot;"/>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_edit_color">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>260</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="inputMask">
    <string comment="&quot;00&quot;" extracomment="&quot;00&quot;"/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sell Price:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Discount:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Color:</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>input_edit_name</tabstop>
  <tabstop>input_edit_merk</tabstop>
  <tabstop>input_edit_sell_price</tabstop>
  <tabstop>input_edit_discount</tabstop>
  <tabstop>input_edit_color</tabstop>
  <tabstop>button_edit_produk</tabstop>
  <tabstop>input_edit_row</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
